src/lwc/utilPageLevelMessage/utilPageLevelMessage.js

```diff
@@ -65,7 +65,7 @@ export default class utilPageLevelMessage extends LightningElement {
     }
 
     get subtitleClass() {
-        let classNames = 'slds-p-top_x-small ';
+        let classNames = 'slds-p-horizontal_medium slds-p-vertical_xx-small ';
```

Comment:
This is shared component, so if the css is changed, every other component using it must be retested. Can we set css outside of the `utilPageLevelMessage` component?

Reply:
Add css override api attribute to not replace the default version